### PR TITLE
English: blog intercom launcher

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
http://blog.learningspaces.io/customizing-the-intercom-launcher/

### CODE REVIEW

File: source/blog/en/2017-01-04-customizing-the-intercom-launcher.html.markdown [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+###### Intercom's missing ready callback
+
+The Intercom [JavaScript API](https://docs.intercom.com/configure-intercom-for-your-product-or-site/customize-the-intercom-messenger/the-intercom-javascript-api) is lacking a `ready` callback, something I've missed from the beginning. There is an `Intercom.boot` flag to test if it's booted, but you still have to watch/poll it (like the example above). I've been talking to Sean, a really helpful Customer Support Engineer at Intercom, about this and he said:
+
+> I'll tag this as a feature request, maybe we can look at getting it added at some point.
+
+---
+
+I hope this helps someone! If you have questions or improvements, please contact us on [Twitter](https://twitter.com/DefactoSoftware) or [Intercom](javascript:Intercom('show')) :)
```
Comment (fatboypunk) [inline source/blog/en/2017-01-04-customizing-the-intercom-launcher.html.markdown]: does this still work?

Comment (RonaldDijkstra) [inline source/blog/en/2017-01-04-customizing-the-intercom-launcher.html.markdown]: The intercom show is same as: www.defacto.nl/contact direct chatten

Comment (fatboypunk) [inline source/blog/en/2017-01-04-customizing-the-intercom-launcher.html.markdown]: Is that also customized?

**Comment by RonaldDijkstra** [comment]
@sn3p do you know?

File: source/blog/en/2017-01-04-customizing-the-intercom-launcher.html.markdown (hunk 1 above)
Comment (RonaldDijkstra) [inline source/blog/en/2017-01-04-customizing-the-intercom-launcher.html.markdown]: No..

**Pull request merged**